### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by itexpert120**

### CODE REVIEW

File: apps/builddao/widget/components/navigation/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,450 @@
+const { Button } = VM.require("buildhub.near/widget/components") || {
```
Comment (elliotBraem) [inline apps/builddao/widget/components/navigation/Navbar.jsx:1]: Let's just move this to components.Navbar

File: apps/builddao/widget/home/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+          >
+            See Activity
+          </Button>
+          <Phrase>
+            Explore the public feed without login.
+            <br />
+            No passphrases, no crypto required.
+          </Phrase>
+        </div>
+        <Stats>
```
Comment (elliotBraem) [inline apps/builddao/widget/home/Hero.jsx:246]: Let's comment these stats out

File: apps/builddao/widget/app.jsx [hunk 3]
```diff
@@ -40,6 +40,170 @@ const Root = styled.div`
...
+      format("woff2");
+  }
+  @font-face {
+    font-family: InterVariable;
+    font-style: italic;
+    font-weight: 100 900;
+    font-display: swap;
+    src: url("https://rsms.me/inter/font-files/InterVariable-Italic.woff2?v=4.0")
+      format("woff2");
+  }
```
Comment (elliotBraem) [inline apps/builddao/widget/app.jsx:206]: I see fontFamily in some of the children -- any way we can just have that defined here for p, h1, h2, or whatever are using it

Comment (itexpert120) [reply apps/builddao/widget/app.jsx:206]: it depends on where it is being used. But I have globally used Inter font so it looks really nice.

File: apps/builddao/widget/home/Hero.jsx (hunk 2 above)
Comment (itexpert120) [reply apps/builddao/widget/home/Hero.jsx:246]: Sure

File: apps/builddao/widget/components/navigation/Navbar.jsx (hunk 1 above)
Comment (itexpert120) [reply apps/builddao/widget/components/navigation/Navbar.jsx:1]: Ok

**Pull request merged**